servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ClientEffectiveStrategyTest.java

```diff
@@ -227,6 +228,7 @@ static Stream<Arguments> casesSupplier() {
         return arguments.stream();
     }
 
+    @Disabled // Disabled due to continued flakiness. See issue #2245 for more details.
```

Comment:
@bryce-anderson This test is very important. It helps to validate that we correctly compute the effective strategy based on all default filters and modules bundled together by default. If we disable it completely, we open a risk to missing a bug that will have a significant performance hit (for example, if we add a filter with non-overridden `requireOffloads()` method). We had that in the past.

Instead of disabling all of them, consider using `assumeThat(...)` statement to disable (with comment) only those test cases that are flaky based on #2245 records. That will help us to make sure the effective strategy is computed correctly if at least some APIs are validated.